apps/service-portal/src/store/modules.ts

```diff
@@ -38,7 +37,6 @@ export type ModuleKeys =
 export const featureFlaggedModules: ModuleKeys[] = [
   'documentProvider',
   'education',
-  'educationDegree',
   'educationLicense',
```

Comment:
Just wondering, why declare separate modules for each of these education sections instead of just including them all under the education module?

Reply:
That's a good question! I'm not sure what the preferred method of working with modules is for the service portal, this is how I was told how it's done and it worked so I didn't question it 😄

Reply:
Each module can declare multiple routes (and even internal routing) so the idea was that elements like these should all be grouped together under education, this might be a good topic for the next discipline meeting so lets bring it up then 😀